src/app/services/reducers/permissions-reducer.ts

```diff
@@ -11,15 +11,21 @@ export function scopes(state = initialState, action: IAction): any {
   switch (action.type) {
     case FETCH_SCOPES_SUCCESS:
       return {
-        ...state,
```

Comment:
>...state [](start = 8, length = 8)

wouldn't this still be good to have in case the state object ever adds new items.